src/app/app.routing.module.ts

```diff
@@ -4,23 +4,21 @@ import { Routes, RouterModule } from '@angular/router';
 import { AuthGuard, UnauthGuard } from 'ngx-prx-styleguide';
 
 import { AuthorizationComponent } from './authorization/authorization.component';
-import { HomeComponent} from './home/home.component';
+import { HomeComponent } from './home/home.component';
 import { LoginComponent } from './login/login.component';
+import { TokenAuthComponent } from './tokenauth/tokenauth.component';
 
 const routes: Routes = [
   { path: 'permission-denied', component: AuthorizationComponent },
-  { path: '',          component: HomeComponent,  canActivate: [AuthGuard] },
-  { path: 'login',     component: LoginComponent, canActivate: [UnauthGuard] }
+  { path: '', component: HomeComponent, canActivate: [AuthGuard] },
+  { path: 'login', component: LoginComponent, canActivate: [UnauthGuard] },
+  { path: 'tokenauth', component: TokenAuthComponent, canActivate: [UnauthGuard] }
```

Comment:
Hmm... yeah, looks like it's a workaround for logging in for Cypress tests.  So it would be needed in prod, if we want to run smoketests there or something.

I'm going to say we leave it for now, but hopefully factor it out someday (doing a "real" login via id.prx.org, then opening Grove is probably a better test that things work).